src/schematools/types.py

```diff
@@ -910,7 +898,7 @@ def from_file(cls, filename: str):
 
     @classmethod
     def from_dict(cls, obj: Dict[str, Any]):
-        """ Parses given dict and validates the given schema """
+        """Parses given dict and validates the given schema"""
```

Comment:
When you rebase this branch on master (force push with lease is fine inside your own branch!), you'll find I've fixed all these black formatting errors.